OmiseSDK/3DS/NetceteraThreeDSController.swift

```diff
@@ -231,43 +281,41 @@ MIIDbzCCAlegAwIBAgIJANp1aztdBEjBMA0GCSqGSIb3DQEBCwUAME4xCzAJBgNVBAMMAmNhMQ4wDAYD
         self.receiver = receiver
         self.transaction = transaction
 
-//        DispatchQueue.main.async {
-            do {
-                try transaction.doChallenge(
-                    challengeParameters: challengeParameters,
-                    challengeStatusReceiver: receiver,
-                    timeOut: 5,
-                    inViewController: viewController
-                )
-            } catch {
-                print("doChallenge error: \(error)")
-                print("-")
-            }
-//        }
+        do {
+            try transaction.doChallenge(
+                challengeParameters: challengeParameters,
+                challengeStatusReceiver: receiver,
+                timeOut: 5,
```

Comment:
Where does `5` come from?

Reply:
Timeout is from Netcetera example - "The minimum timeout interval is defined to be 5 minutes" https://3dss.netcetera.com/3dssdk/doc/2.10.0/ios-sdk-api